content/en/blog/identity-hack/index.md

```diff
@@ -0,0 +1,76 @@
+
+To seize these opportunities, OP Lab and EU funded Powered by Blockchain project organized in collaboration a **seminar day and online hackathon** aiming to explore the disruptive potential of SSI technologies and institutional decentralized finance.  
+
+
+## Kicking Off the Hackathon Challenge 
+
+The main theme of the event was ***Digital Identity as an Enabler for Open Decentralized Finance***. The emergence of SSI technologies has opened new opportunities by enabling verifiable credentials for identity and data verification. The [hackathon](https://new.ultrahack.org/hackathons/digital-identity-hack) and seminar day aimed to educate participants on how to leverage these technologies to transform the world of finance. The live event, bringing together tech professionals, students, and companies, was recently held in Oulu, Finland and streamed online for a wider audience to participate. 
+
+## Challenge Objectives and Key Topics 
+
+The objective of the hackathon challenge was to **develop financial products and services that are transparent, open, and accessible to anyone on the Internet**. To achieve this, the organizers believe that SSI technologies may offer great tools for realizing the goal. 
+
+Key topics covered during the challenge presentation included combining verifiable credentials with blockchain technology, checking identity and other data attributes in transactions, controlling transactions automatically through smart contracts based on verifiable data, and creating confidential private transactions while verifying identity data. 
+
+Participants were also presented with example approaches to the challenge, such as how to verify financial compliance like Know Your Customer (KYC) and Anti-Money Laundering (AML), how to check company employee credentials to proceed with company transactions, and how to verify investor accreditation. 
+
+## Inspirational Case Studies 
+
+During the event, several intriguing proof-of-concept case examples were highlighted, providing valuable insights into the promising potential of this innovative and uncharted domain. 
+
+**Onyx by J.P. Morgan** has been exploring the use of verifiable credentials and blockchain technology in foreign exchange transactions through Project Guardian. This concept has demonstrated that by utilizing tokenized forms of cash and proper authorization, it is possible to create decentralized financial services with the necessary safeguards in place. 
+
+* Project Guardian - [https://www.jpmorgan.com/onyx/content-hub/institutional-defi.htm](https://www.jpmorgan.com/onyx/content-hub/institutional-defi.htm)
+
+
+{{< imgproc project-jupiter.png Fit "939x649" />}}
```

Comment:
if you rename the image file with prefix "cover", e.g. cover-project-jupiter.png, it will be shared as the cover image to SoMe shares

Reply:
Added cover photo